.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "hpack-test-case"]
+	path = hpack-test-case
+	url = git@github.com:http2jp/hpack-test-case
```

Comment:
quick question for @normanmaurer . Do we need to add this to `NOTICE.txt`? It's only used for tests.